core-web/libs/template-builder/src/lib/components/template-builder/store/template-builder.store.ts

```diff
@@ -350,6 +351,8 @@ export class DotTemplateBuilderStore extends ComponentStore<DotTemplateBuilderSt
 
         if (!container) return state;
 
+        const containerReference = getContainerReference(container);
```

Comment:
If you do this in the container selector we don't have to add it anywhere else.

Reply:
Yes! Going to do it that way